server/api/validators/geojson.spec.js

```diff
@@ -6,6 +6,113 @@ const geoJsonValidators = require('./geojson');
 
 describe('GeoJSON validators', function() {
 
+  describe('bboxString', function() {
+
+    const validateBboxString = geoJsonValidators.bboxString;
+
+    it('should be a function', function() {
+      expect(validateBboxString).to.be.a('function');
+    });
+
+    it('should create a validator function', function() {
+      expect(validateBboxString()).to.be.a('function');
+    });
+
+    it('should add no error for a valid bounding box', async function() {
+      const err = await validate('10,20,30,40');
+      expect(err).to.equal(undefined);
+    });
+
+    it('should add an error if the bounding box is not a string', async function() {
+      const err = await validate(10203040);
+      expect(err).not.to.equal(undefined);
+      expectErrors(err, [
```

Comment:
Why test for `err` to not be `undefined` when the following expectation should only pass if `err` is defined ?
If the `expectErrors` test fails, I suppose it will say something like : `"Expected undefined to equal [blabla]"`, which will indicate that `err` was `undefined`, right ?

(Note : this also applies to lines 43, 59 and 88.)

Reply:
It's not very useful, indeed. I will move the assertion to `expectErrors` so that it's only written once.